### PR TITLE
Allow to provide multiple values of property when build search query.

### PR DESCRIPTION
**Pull request by OlgaCh**
Closes https://github.com/sentinelsat/sentinelsat/issues/317

Enables an option to provide values for same parameter as a set.

### CODE REVIEW

**Comment by OlgaCh** [comment]
@j08lue could you review this please?
Sorry for messy commit history. I've forget to run black before submitting PR and then have some issues dealing with it's warnings during the Travis build.

**Comment by valgur** [comment]
Should also add a mention of this new functionality in documentation, but this can be addressed in another PR.

File: sentinelsat/sentinel.py [hunk 1]
```diff
@@ -209,7 +209,9 @@ def format_query(area=None, date=None, raw=None, area_relation="Intersects", **k
             date_attrs = ["beginposition", "endposition", "date", "creationdate", "ingestiondate"]
             if attr.lower() in date_attrs:
                 # Automatically format date-type attributes
-                if isinstance(value, string_types) and " TO " in value:
+                if isinstance(value, set):
```
Comment (j08lue) [inline sentinelsat/sentinel.py]: @Fernerkundung and @valgur - only `set` should be coerced to `OR`? Not `tuple` and `list`, too?

Comment (j08lue) [reply sentinelsat/sentinel.py]: I guess it makes most sense for `set`, because it has unique elements and is unordered.

Comment (Fernerkundung) [reply sentinelsat/sentinel.py]: In the current state only sets are coerced as `OR`. Everything else gets parsed as `START TO END`.

Do we also want to support: `START1 TO END1 OR START2 TO END2`? That would require some changes to the logic. However, I can't see a compelling use case for this that can't be solved with 2 queries.

Comment (j08lue) [reply sentinelsat/sentinel.py]: no, fine this way then.

**Pull request merged**